### PR TITLE
Added: Change User to Service / Service to User

### PR DESCRIPTION
**Pull request by CHDAFNI-MSFT**
clarified on capabilities of the PTN@microsoft.com dept.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/manage-phone-numbers-for-your-organization/phone-number-management-for-the-u-s.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ For more information, see [Manage phone numbers for your organization](manage-ph
...
-|Change a user's number to a service number.  <br/> |[Send us](mailto:ptn@microsoft.com) your PIN and the list of user numbers you want to convert to service numbers. To set up a PIN, go [here](/skypeforbusiness/what-are-calling-plans-in-office-365/set-your-pin-for-transferring-numbers-to-a-new-service-provider).  <br/> |
+|Change a user's number to a service number or a service number to a user's number  <br/> |[Send us](mailto:ptn@microsoft.com) your PIN and the list of numbers you want to convert. To set up a PIN, go [here](/skypeforbusiness/what-are-calling-plans-in-office-365/set-your-pin-for-transferring-numbers-to-a-new-service-provider).  <br/> |
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/phone-number-management-for-the-u-s.md:37]:
```suggestion
|Change a user's number to a service number or a service number to a user's number  <br/> |[Send us](mailto:ptn@microsoft.com) your PIN and the list of numbers you want to convert. To set up a PIN, go [here](/skypeforbusiness/what-are-calling-plans-in-office-365/set-your-pin-for-transferring-numbers-to-a-new-service-provider).  <br/> |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**